src/main/kotlin/uk/gov/justice/digital/hmpps/incidentreporting/jpa/IncidentEvent.kt

```diff
@@ -0,0 +1,64 @@
+package uk.gov.justice.digital.hmpps.incidentreporting.jpa
+
+import jakarta.persistence.CascadeType
+import jakarta.persistence.Column
+import jakarta.persistence.Entity
+import jakarta.persistence.FetchType
+import jakarta.persistence.GeneratedValue
+import jakarta.persistence.GenerationType
+import jakarta.persistence.Id
+import jakarta.persistence.OneToMany
+import org.slf4j.Logger
+import org.slf4j.LoggerFactory
+import uk.gov.justice.digital.hmpps.incidentreporting.dto.EventDetail
+import java.io.Serializable
+import java.time.LocalDateTime
+
+@Entity
+class IncidentEvent(
+
+  @Id
+  @GeneratedValue(strategy = GenerationType.IDENTITY)
+  val id: Long? = null,
+
+  @Column(nullable = false, unique = true, length = 25)
+  val eventId: String,
+  val eventDateAndTime: LocalDateTime,
+  val prisonId: String,
+  var summary: String,
+  var eventDetails: String,
```

Comment:
what's the difference between summary and details?

Reply:
Yes not sure there would be one, I've kept it as NOMIS has the concept of title and description and I've reproduced for Event.  Not sure its worth it